### PR TITLE
update dependency list

### PR DESCRIPTION
**Pull request by pfuehrlich-pik**
I just ran `renv::dependencies()` to auto-detect dependencies. I manually added `goxygen` which is not used in R files, but it is mentioned in the readme to generate documentation. The full command I used before copy-pasting:
```
cat(paste0('"', sort(unique(renv::dependencies()$Package)), '"', collapse = ",\n"))
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,20 +99,46 @@ variable. After that you can run the following lines of code:
...
+          "lusweave",
+          "magclass",
+          "magpie4",
+          "mip",
+          "mrremind",
+          "mrvalidation",
+          "optparse",
+          "plotly",
+          "quitte",
+          "remind",
```
Comment (mikapfl) [inline README.md:132]: Do we still depend on remind as well as remind?

**Comment by mikapfl** [comment]
There is another dependency list in https://github.com/remindmodel/remind/blob/develop/tutorials/1_GettingREMIND.md, probably it also needs updating? Or both need merging?

File: README.md (hunk 1 above)
Comment (johanneskoch94) [reply README.md:132]: Nope, #785

Comment (pfuehrlich-pik) [reply README.md:132]: Did you already test the suggestion from the issue? Can I just add your suggestion to this PR?

Comment (johanneskoch94) [reply README.md:132]: I have not tested it, but I think it will work.

Comment (pfuehrlich-pik) [reply README.md:132]: Could you test it? I think this is much quicker than if I try to find out how to call the function now

Comment (johanneskoch94) [reply README.md]: I tested it, looks fine to me.

**Comment by pfuehrlich-pik** [comment]
I replaced `remind::` with `remind2::` as suggested by Johannes, so the remind dependency is gone now

**Comment by pfuehrlich-pik** [comment]
I think we can merge now, but I still need approval :wink:

**Review by johanneskoch94 — APPROVED** [review]

**Pull request merged**